package/com.unity.formats.usd/Tests/Editor/ImportHelpersTests.cs

```diff
@@ -4,95 +4,127 @@
         {
-            var dummyUsdPath = CreateTmpUsdFile(fileName);
-            var scene = ImportHelpers.InitForOpen(dummyUsdPath);
-            scene.Write("/root", new XformSample());
-            scene.Write("/root/sphere", new SphereSample());
-            scene.Save();
-            return scene;
+            var scene = CreateTestUsdScene();
+            scene.Close();
+            Assert.Throws<System.NullReferenceException>(() => ImportHelpers.ImportAsTimelineClip(scene, GetPrefabPath()));
         }
 
         [Test]
         public void ImportAsPrefabTest_ContentOk()
         {
-            var scene = CreateTestAsset("dummyUsd.usda");
+            var scene = CreateTestUsdScene();
             var assetPath = ImportHelpers.ImportAsPrefab(scene, GetPrefabPath());
-            Assert.IsNull(scene.Stage, "Scene was not closed after import.");
 
             Assert.IsTrue(File.Exists(assetPath));
-            var allObjects = AssetDatabase.LoadAllAssetsAtPath(assetPath);
-            Assert.NotZero(allObjects.Length);
-            bool playableAssetFound = false;
-            int goCount = 0;
-            int materialCount = 0;
-            foreach (Object thisObject in allObjects)
-            {
-                string myType = thisObject.GetType().Name;
-                if (myType == "UsdPlayableAsset")
-                {
-                    playableAssetFound = true;
-                }
-                else if (myType == "GameObject")
-                {
-                    goCount += 1;
-                }
-                else if (myType == "Material")
-                {
-                    materialCount += 1;
-                }
-            }
+            var usdAsObjects = AssetDatabase.LoadAllAssetsAtPath(assetPath);
 
-            Assert.IsTrue(playableAssetFound, "No PlayableAssset was found in the prefab.");
-            Assert.AreEqual(2, goCount, "Wrong GameObjects count in the prefab.");
-            // The 3 default materials + 1 material per meshRender
-            Assert.AreEqual(4, materialCount, "Wrong Materials count in the prefab");
+            // ExpectedGameObjectCount: The Root GameObject + Sphere GameObject added
+            // ExpectedPrimSourceCount: Root Source + Sphere Source
+            // ExpectedMaterialCount: The 3 default materials + 1 material from Sphere meshRender
+            EditorImportAssert.IsValidImport(usdAsObjects, expectedGameObjectCount: 2, expectedPrimSourceCount: 2, expectedMaterialCount: 4);
         }
 
         [Test]
         public void ImportAsTimelineClipTest_ContentOk()
         {
             // Import as timeline clip should not create a hierarchy, only the root and the playable
-            var scene = CreateTestAsset("dummyUsd.usda");
+            var scene = CreateTestUsdScene();
             var assetPath = ImportHelpers.ImportAsTimelineClip(scene, GetPrefabPath());
-            Assert.IsNull(scene.Stage, "Scene was not closed after import.");
```

Comment:
As above, is there a reason not to keep this?

Reply:
This part was tested as a part of a previous testcase ImportAsTimelineClipTest_SceneClosedAfterImport
No need to check for this again here